runWeathervane.pl

```diff
@@ -449,9 +447,16 @@ sub forceLicenseAccept {
     $sshMountString = "-v $homeDir/.ssh:/root/.ssh";
 }
 
-my $retRef = parseConfigFile($configFile);
+my $dockerNamespace;
+# Determine whether the dockerNamespace is set in the command-line
+# options that are getting passed to the runHarness
+if ($wvCommandLineArgs =~ /\-\-dockerNamespace(\=|\s)([^\s]*)(\s|$)/) {
```

Comment:
Would it be valid to have multiple spaces following dockerNamespace when not using '='?
--dockerNamespace  (multiple spaces here)   myName
if so, 1st capturing group could maybe adjust (\=|\s) to (\=|\s+)    ?

Reply:
Actually, the way that perl puts arguments into ARGV ignores extra spaces, and the way that runWeathervane.pl puts them back together into a single string, ensures that there will never be multiple consecutive spaces in $wvCommandLineArgs.